gps/gpsparser.go

```diff
@@ -95,6 +95,8 @@ func (parser *Parser) Parse(sentence string) (Fix, error) {
 		fix.Longitude = findLongitude(fields[5], fields[6])
 		fix.Speed = findSpeed(fields[7])
 		fix.Heading = findHeading(fields[8])
+		date := findDate(fields[9])
```

Comment:
Out of the three supported sentence types (the cases in this `switch` block), only `RMC` contains the date. I was thinking of maybe adding in a type alias such as `type FixType string` and adding it as a field on the `Fix` struct to make it easier for consumers to see if they can meaningfully grab the date out of the `Fix` instance or not:

```go
if fix.Type == gps.RMC {
    month := fix.Time.Month()
}
```

If this sounds good, please let me know and I can add it.